README.md

```diff
@@ -5,18 +5,13 @@
   <a href="https://goreportcard.com/report/github.com/golang/dep"><img src="https://goreportcard.com/badge/github.com/golang/dep" /></a>
 </p>
 
-<i>Would you like to see your company name here? <a
-href="https://github.com/golang/dep/issues/2165">We're looking for a stable source of
-funding.</a></i>
-
 ## Dep
 
 `dep` is a dependency management tool for Go. It requires Go 1.9 or newer to compile.
 
-`dep` was initially developed to experiment with a new dependency management system
-for Go; but, as of Go 1.11, **the Go project has officially adopted a different approach, 
-based on the concept of Modules**, which is described in a [series of posts](https://blog.golang.org/using-go-modules) 
-published on [blog.golang.org](https://blog.golang.org).
+Please note that Dep was an official experiment to implement a package manager
```

Comment:
I would perhaps add some visual emphasis  to make this clearly stand out, something like:

> **`** NOTE: **`** Dep was an official experiment to ...

Reply:
Done